velox/exec/fuzzer/AggregationFuzzerBase.cpp

```diff
@@ -300,14 +306,28 @@ std::vector<RowVectorPtr> AggregationFuzzerBase::generateInputData(
 std::vector<RowVectorPtr> AggregationFuzzerBase::generateInputDataWithRowNumber(
     std::vector<std::string> names,
     std::vector<TypePtr> types,
-    const CallableSignature& signature) {
+    const CallableSignature& signature,
+    const bool& hasKRangeFrameBound) {
   names.push_back("row_number");
   types.push_back(BIGINT());
 
   auto generator = findInputGenerator(signature);
 
   std::vector<RowVectorPtr> input;
   auto size = vectorFuzzer_.getOptions().vectorSize;
+
+  int64_t offsetStart;
```

Comment:
Is it possible to delay adding these types and column values at the point where the frame is generated ? That would make it more clean imo.